gallery/src/components/NxButton/NxButtonIconOnlyExample.tsx

```diff
@@ -11,9 +11,12 @@ import { NxButton, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
 
 const NxButtonIconOnlyExample = () =>
   <div className="nx-btn-bar">
-    <NxButton variant="icon-only"><NxFontAwesomeIcon icon={faExclamationTriangle}/></NxButton>
-    <NxButton variant="icon-only"><NxFontAwesomeIcon icon={faEdit}/></NxButton>
-    <NxButton variant="icon-only"><NxFontAwesomeIcon icon={faSave}/></NxButton>
+    <NxButton variant="icon-only"><NxFontAwesomeIcon aria-hidden={false}
+                                                     aria-label='alert'
```

Comment:
Should the label be on the icon or the button? I'll admit it works better than I expected as-is; I was worried it would announce it as "alert. image. button". But it skips the "image" part.

Reply:
020855aceda0bd911601c65b1fb5f0a82e3f3bb0